Server/mods/deathmatch/logic/luadefs/CLuaAccountDefs.cpp

```diff
@@ -513,6 +515,31 @@ int CLuaAccountDefs::RemoveAccount(lua_State* luaVM)
     return 1;
 }
 
+int CLuaAccountDefs::SetAccountSerial(lua_State* luaVM)
```

Comment:
Please use the new argument parser here (https://github.com/multitheftauto/mtasa-blue/wiki/Lua-API#how-to-add-a-new-lua-definition)

Reply:
I hadn't seen the update, I switched to the new standard 62ebf06